core-library/src/main/java/org/silverpeas/core/index/indexing/model/FieldDescription.java

```diff
@@ -101,11 +103,22 @@ public boolean isStored() {
     return stored;
   }
 
+  public Date getStartDate() {
+    return startDate;
+  }
+
+  public Date getEndDate() {
+    return endDate;
+  }
+
   /**
    * All the attributes are private and final.
    */
   private final String content;
   private final String lang;
   private final String fieldName;
   private final boolean stored;
-}
+  private final boolean basedOnDates;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)